pkg/bao/bao.go

```diff
@@ -69,6 +69,25 @@ func Find[ModelT any](ctx context.Context, db bun.IDB, queryFn func(q *bun.Selec
 	return model, nil
 }
 
+func FindFirst[ModelT any](ctx context.Context, db bun.IDB, queryFn func(q *bun.SelectQuery)) (*ModelT, error) {
+	model := new(ModelT)
```

Comment:
Why are we using new(ModelT) here instead of the pattern followed in the other find funcs?